packages/components/Atoms/Icon/icons/index.js

```diff
@@ -86,6 +86,7 @@ import './upload'
 import './user'
 import './view'
 import './webinar'
+import './whatsapp'
```

Comment:
Adding Whatsapp icon here is not so bad, but note projects can register their own custom icons by placing them in `/assets/ripple-icon` folder - See https://github.com/dpc-sdp/ripple/blob/f177592f80561824b30350cebf48b9c74c0dd7bc/examples/basic-examples/assets/ripple-icon/custom_icon_search.js  And webpack loader config https://github.com/dpc-sdp/ripple/blob/develop/packages/ripple-nuxt-ui/lib/module.js

Reply:
Hey @dylankelly! thanks for the links - I did try to add it to /assets/ripple-icon; it worked fine if referenced inside my project, under /components for example, but didn't render when referenced from inside a ripple dependency. I was probably missing something in webpack, so will have a look at that.